app.js

```diff
@@ -0,0 +1,221 @@
+const token = require("./token.json");
+const Discord = require("discord.js");
+var fs = require("fs");
+var _ = require("lodash");
```

Comment:
are you using lodash?
if not, remove it

Reply:
Yes, I'm using it for capitalizing the category names so that there will be no errors while accessing the data. or else user have to write the exact name as it is in data.